Functions/Initialization/xASL_init_InitializeMutex.m

```diff
@@ -64,16 +64,80 @@
 %% --------------------------------------------------------
 %% 2) Initialize mutex object
 x.mutex = xASL_GoNoGo(x.dir.LockDir);
+
+%% 3) Check if this module is locked by another process
+bLocked = 0;
+
 if ~x.mutex.Lock(x.settings.MUTEXID)
-    fprintf('Also, check that there is no filesystem permission issue\n');
 	fprintf(2,['ERROR in module_' x.ModuleName ': mutex is locked: %s in %s\n'], x.settings.MUTEXID, x.dir.LockDir);
-	fprintf('This means that this module is currently being parallel processed by another Matlab instance/worker\n');
+	fprintf('\n');
+    fprintf('This means that this module is currently being parallel processed by another Matlab instance/worker\n');
 	fprintf('If this is not the case, the locked folder needs to be removed before proceeding\n');
 	fprintf('Also, check that there is no filesystem permission issue\n');
     fprintf('Otherwise this error can be ignored\n');
-	return;
-end
+    fprintf('\n');
+
+    bLocked = 1;
+else
+    
+    %% 4) Check if any module for this subject is locked
+    if ~strcmpi(x.ModuleName, 'population')

```

Comment:
For the population module. I propose that it won't even try to lock the directory if it finds out it runs in parallel mode...

Reply:
But all this still done elsewhere.

Reply:
True, this is just a sanety check. The population module won't run if ExploreASL has multiple workers.

Reply:
OK. As a second level sanity check, this is fine. Could you please add that inside the code in a comment, so that we understand this later as well?

Reply:
Done